src/components/Carousel/Carousel.scss

```diff
@@ -0,0 +1,25 @@
+.Carousel {
+  display: flex;
+  align-items: center;
+  .Carousel-listWrapper {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    overflow-x: hidden;
+  }
+  .Carousel-list {
+    display: flex;
```

Comment:
Do `Carousel-list` and `Carousel-listWrapper` not inherit `display: flex` from their parents, or do you need to manually specify it here?

Reply:
No, they won't be inheriting by default. It has to be specified manually.